src/content/guides/tree-shaking.md

```diff
@@ -57,6 +58,24 @@ export function cube(x) {
 }
 ```
 
+You could need to set the development mode to be sure that bundle is not minified:
+
+__webpack.config.js__
+
+``` diff
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist')
+- }
++ },
++ mode: "development"
```

Comment:
Sorry for being late to the party but, is it necessary to specify entry and output when they have default values already?

Also, I think that I would add the mode by CLI and put a small warning on the guide maybe.

Reply:
You are right if you take the single page guide alone, but in the contest of the get started guide where you  build the concept step by step you already have the webpack.config.js